diffusers/Asymetric Tiling.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>Replace with for more context and easier to follow along if a user is landing on this for the first time</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Asymmetric tiling

Stable Diffusion is not trained to generate seamless textures. However, you can use this simple script to add tiling to your generation. This script is contributed by&nbsp;[alexisrolland](https://github.com/alexisrolland). See more details in the&nbsp;[this issue](https://github.com/huggingface/diffusers/issues/556).
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/525/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>